content/release_notes_client.md

```diff
@@ -12,3 +12,16646 @@ release_notes = "chef"
     parent = "release_notes"
     weight = 20
 +++
+
```

Comment:
All of this should be deleted since we moved the RN for client to seamless RN. See #3422